src/graphics.jl

```diff
@@ -7,20 +7,19 @@ function printrow end
 function Base.print(io::IO, c::GraphicsArea)
     for row in 1:nrows(c)
         printrow(io, c, row)
-        row < nrows(c) && print(io, '\n')
+        println(io)
```

Comment:
I believe it's deliberate to not add a new line at the end because Julia REPL will insert one. For example, with this PR there are two empty lines at the end.

<img src="https://user-images.githubusercontent.com/8684355/132273645-ce250a34-4917-4ae3-958b-cc0026f0748a.png" alt="preview.png" width="600">

Reply:
Fixed.